File: tools/bench/answers/python_haiku/s3/15_wordlens.py
sentence = "pack my box with five dozen liquor jugs"
words = sentence.split()
longest = words[0]
for word in words:
    if len(word) > len(longest):
        longest = word
total_len = 0
for word in words:
    total_len += len(word)
avg_len = int(total_len / len(words))
print(longest)
print(avg_len)

File: tools/bench/answers/rust_haiku/s3/15_wordlens.rs
fn main() {
    let sentence = "pack my box with five dozen liquor jugs";
    let words: Vec<&str> = sentence.split_whitespace().collect();

    let mut longest = "";
    for &word in &words {
        if word.len() > longest.len() {
            longest = word;
        }
    }

    let mut total_len = 0;
    for &word in &words {
        total_len += word.len();
    }
    let avg_len = total_len / words.len();

    println!("{}", longest);
    println!("{}", avg_len);
}
